src/skin/legacyskinparser.cpp

```diff
@@ -102,7 +102,7 @@ ControlObject* controlFromConfigKey(const ConfigKey& key, bool bPersist,
     }
     // Don't warn if the control doesn't exist. Skins use this to create
     // controls.
-    ControlObject* pControl = ControlObject::getControl(key, false);
+    ControlObject* pControl = ControlObject::getControl(key, ControlFlag::NoWarnIfMissing);
```

Comment:
This will become ControlFlag::None after my suggested changes.

Reply:
Instead of Implicit disable warnings, we can introduce another function that is called getOrCreateControl() or something.